projects/media-viewer/src/lib/shared/directives/integer-input-directive.ts

```diff
@@ -0,0 +1,23 @@
+import { Directive, ElementRef, HostListener, Self } from '@angular/core';
```

Comment:
is this actually needed? would <input type="number"> not do the same job?

Reply:
I wanted to prevent the user from using the up/down arrows when inside the page number field so thought I'd add this, however on reflection I will remove this directive and simply add css to remove the up/down arrows that appear inside a number input field as there are already page up and down buttons on the toolbar.